docs/references/ic-interface-spec.md

```diff
@@ -2293,6 +2295,26 @@ Indicates various information about the canister. It contains:
 
 Only the controllers of the canister or the canister itself can request its status.
 
+#### Memory Metrics {#ic-canister_status-memory_metrics}
+
+    * `heap_size`: Represents the Wasm memory usage of the canister, i.e. the heap memory used by the canister's WebAssembly code.
+
+    * `stable_size`: Represents the stable memory usage of the canister, which is the memory that persists across canister upgrades.
+
+    * `global_size`: Represents the memory usage of the global variables that the canister is using.
+
+    * `wasm_binary_size`: Represents the memory occupied by the Wasm binary that is currently installed on the canister.
```

Comment:
Let's point out that this is the size of the binary uploaded via `install_code` or `install_chunked_code`, e.g., the compressed size if the uploaded binary is gzipped.